openproblems/tasks/_batch_integration/batch_integration_embed/api.py

```diff
@@ -1,16 +1,12 @@
-from ....data.sample import load_sample_data
 from ....tools.decorators import dataset
-
-import numpy as np
+from .._common import api
 
 
 def check_dataset(adata):
     """Check that dataset output fits expected API."""
+    api.check_dataset(adata)
 
     assert "X_uni_pca" in adata.obsm
-    assert "batch" in adata.obs
-    assert "labels" in adata.obs
-    assert "log_normalized" in adata.layers
     assert "organism" in adata.uns
     assert adata.uns["organism"] in ["mouse", "human"]
```

Comment:
Organism check should also be put into the common API, no? @danielStrobl ? This is for CC cons metric... it's only run on embeddings here, but could also be run on feature outputs... and maybe it's good practice to be consistent for datasets across the subtasks.

Reply:
Fine by me to require in all subtasks